### PR TITLE
CREATE / users

### PR DESCRIPTION
**Pull request by iosifboanca**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-  "name": "tsa-node-project",
+  "name": "concedii",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "start": "nodemon src/main.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "repository": {
```
Comment (andreicioromila) [inline package.json:10]: Please try not to commit changes to `package.json` and other common files in feature branches. Any _necessary_ changes should be done on a separate branch (except for installing new deps which are specific to the feature being developed)

File: db/users.json [hunk 2]
```diff
@@ -1,9 +1,2 @@
...
-    {
-        "userId": 0,
-        "first_name": "",
-        "last_name": "",
-        "email": "",
-        "phone": 0
-    }
-]
\ No newline at end of file
+[{ "firstName": "", "lastName": "", "email": "", "phone": "", "userId": 0 }]
```
Comment (MihaiBojescu) [inline db/users.json]: După ultima discuție am stabilit că vom folosi `"id"` în loc de `"userId"`.

File: src/actions/createUser.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+let fs = require("fs");
+let users = require("../../db/users.json")
+
+let createUser = (req, res) => {
+  let name = req.body;
+  if (dataValid(name) === 1) {
+    name.userId = idMax(users)+1;
+    users.push(name);
+    fs.writeFile("db/users.json", JSON.stringify(users), err => {
+      if (err) throw err(console.log("Nu s-a putut suprascrie"));
```
Comment (andreicioromila) [inline src/actions/createUser.js]: check type of `err`. It most probably is an object, which means you can not use it as a function.

Comment (andreicioromila) [inline src/actions/createUser.js]: What happens with the error if you throw here?

**Review by MihaiBojescu — CHANGES_REQUESTED** [review]

**Review by andreicioromila — CHANGES_REQUESTED** [review]

File: doc/getUser.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      **Content:** 
+      ```javascript
+      {
+         error_fname: “Ati introdus prenumele gresit”,
+	    error_lname: “Ati introdus numele gresit”,
+	    error_email: “Ati introdus emailul gresit”,
+	    error_phone: “Ati introdus gresit numarul de telefon”
+      }
+      ```
+
```
Comment (taniafecheta) [inline doc/getUser.md:36]: Trebuie schimbat numele fisierului din getUser in altceva

**Review by taniafecheta — CHANGES_REQUESTED** [review]

**Pull request closed without merging**